docs/attributes-registry/db.md

```diff
@@ -166,6 +166,16 @@
 | `ExecuteJavaScript` | execute_javascript |
 <!-- endsemconv -->
 
+<!-- semconv registry.db(omit_requirement_level,tag=tech-specific-cosmosdb) -->
```

Comment:
Since `rntbd` is not a database, but a protocol, I suggest putting the table into `rpc` folder.

Reply:
if you mean, putting this table in `rpc.md` and `rpc.yaml` then why `rpc`? I think `rntbd` is closer to `http`.

I think its better to have new `md` and `yaml` file for `rntbd` and refer them in `cosmosdb.md`